osmaxx-py/osmaxx/excerptexport/templates/excerptexport/templates/new_excerpt_export.html

```diff
@@ -33,22 +33,22 @@
                         <select class="form-control" name="existing_excerpt.id" id="existing_excerpt.id" size="10" required="required">
                             <optgroup label="&nbsp;{% trans 'Personal excerpts' %} ({{ user.username }})">
                                 {% for excerpt in excerpts.own_private %}
-                                    <option value="{{ excerpt.id }}">{{ excerpt.name }}</option>
+                                    {% include 'excerptexport/partials/excerpt_option.html' with excerpt=excerpt %}
```

Comment:
`with excerpt=excerpt` statement wouldn't explicitly be needed here, should we remove it? (Same is true below).

Alternatively we could do something like:

```
{% with excerpts=excerpts.own_private %}
    {% include 'excerptexport/partials/excerpt_options.html' %}
{% endwith %}
```

and in `excerptexport/partials/excerpt_options.html`:

```
{% for excerpt in excerpts %}
    {% include 'excerptexport/partials/excerpt_option.html' %}
{% endfor %}
```